stl/inc/execution

```diff
@@ -833,8 +833,9 @@ struct _Static_partition_range;
 template <class _RanIt, class _Diff>
 struct _Static_partition_range<_RanIt, _Diff, true> {
     using _Target_diff = _Iter_diff_t<_RanIt>;
-    _Unwrapped_t<_RanIt> _Start_at;
-    using _Chunk_type = _Iterator_range<_Unwrapped_t<_RanIt>>;
+    using _URanIt      = _Unwrapped_t<const _RanIt&>;
+    _URanIt _Start_at;

```

Comment:
It feels more consistent to have all the using declarations together and then declare `_URanIt _Start_at;`

Reply:
I agree that it's generally preferable style to keep types and data members separate. I'd like to make this change, but not in this PR. I've filed #330 to track it, and roughed up a fix in https://github.com/CaseyCarter/STL/tree/execleanup which I'll push after this merges and I've thrown a compiler at it to make sure I didn't throw a typo in there somewhere.